packages/calendar/src/components/ui/date-time-input/DateTextInputTheme.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  defaultStandardTextInputTheme,
+  StandardTextInputTheme
+} from "@stenajs-webui/forms";
+import {
+  CalendarTheme,
+  defaultCalendarTheme
+} from "../calendar/components/CalendarTheme";
+
+export interface DateTextInputTheme {
+  calendar: CalendarTheme;
+  textInput: StandardTextInputTheme;
```

Comment:
Maybe name it `standardTextInput`, so that the name is always the same as the component?